### PR TITLE
Change Ansible installation to use yum

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by yussufsh** [comment]
/hold
Doing last round of testing with `rhel_subscription_password  = "<subscription-password>"`
In meantime would like some review comments if any...
/cc @Prajyot-Parab @bpradipt

File: docs/var.tfvars-doc.md [hunk 1]
```diff
@@ -196,12 +196,17 @@ install_playbook_repo      = "https://github.com/ocp-power-automation/ocp4-playb
...
-These variables can be used when debugging ansible playbooks
+These variables can be used when debugging ansible playbooks.
 ```
 installer_log_level         = "info"
 ansible_extra_options       = "-v"
 ```
 
+This variable can be used to change the repository name for installing ansible package on RHEL.
+```
+ansible_repo_name           = "ansible-2.9-for-rhel-8-ppc64le-rpms"
```
Comment (bpradipt) [inline docs/var.tfvars-doc.md:207]: Do we really need this ?

File: modules/1_prepare/prepare.tf [hunk 2]
```diff
@@ -238,9 +238,9 @@ EOF
     }
 }
 
-resource "null_resource" "bastion_packages" {
+resource "null_resource" "enable_repos" {
     count           = local.bastion_count
-    depends_on      = [null_resource.bastion_init, null_resource.setup_proxy_info, null_resource.bastion_register]
+    depends_on  = [null_resource.bastion_init, null_resource.setup_proxy_info, null_resource.bastion_register]
```
Comment (bpradipt) [inline modules/1_prepare/prepare.tf:243]: format?

File: docs/var.tfvars-doc.md (hunk 1 above)
Comment (yussufsh) [reply docs/var.tfvars-doc.md:207]: In case we want to update in future... let me know if you think that's fine, I will be happy to remove it.

Comment (bpradipt) [reply docs/var.tfvars-doc.md:207]: Will go with your judgement ..

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#pullrequestreview-555351566" title="LGTM">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#pullrequestreview-555361854" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#pullrequestreview-555351566" title="LGTM">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/186#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**